test_sets/ExternalUserConsentStatus_Informing_HMI.txt

```diff
@@ -0,0 +1,9 @@
+./test_scripts/Polices/External_UCS_Informing_HMI/001_ATF_P_GetListofPermissions_with_appID_all_ecs_in_bound_EntitySatus_ON.lua Check_that_SDL_returns_all_params_in_bound_from_OnAppPermissionConsent_notification_Entity_Status_ON
+./test_scripts/Polices/External_UCS_Informing_HMI/002_ATF_P_GetListofPermissions_with_appID_all_ecs_lower_bound_EntitySatus_ON.lua Check_that_SDL_returns_all_params_lower_from_OnAppPermissionConsent_notification_Entity_Status_ON
+./test_scripts/Polices/External_UCS_Informing_HMI/003_ATF_P_GetListofPermissions_with_appID_all_ecs_upper_bound_EntitySatus_ON.lua Check_that_SDL_returns_all_params_upper_from_OnAppPermissionConsent_notification_Entity_Status_ON
+./test_scripts/Polices/External_UCS_Informing_HMI/004_ATF_P_GetListofPermissions_with_appID_all_ecs_in_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_in_bound_from_OnAppPermissionConsent_notification_Entity_Status_OFF
+./test_scripts/Polices/External_UCS_Informing_HMI/005_ATF_P_GetListofPermissions_with_appID_all_ecs_lower_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_lower_from_OnAppPermissionConsent_notification_Entity_Status_OFF
+./test_scripts/Polices/External_UCS_Informing_HMI/006_ATF_P_GetListofPermissions_with_appID_all_ecs_upper_bound_EntitySatus_OFF.lua Check_that_SDL_returns_all_params_upper_from_OnAppPermissionConsent_notification_Entity_Status_OFF
+./test_scripts/Polices/External_UCS_Informing_HMI/007_ATF_P_GetListofPermissions_with_appID_not_stored_ecs.lua Check_that_SDL_returns_empty_array_appID_no_stored_ecs
+./test_scripts/Polices/External_UCS_Informing_HMI/008_ATF_P_GetListofPermissions_no_appID_stored_ecs_present.lua Check_that_SDL_responds_with_values_from_ecs_no_appID
+./test_scripts/Polices/External_UCS_Informing_HMI/009_ATF_P_GetListofPermissions_no_appID_not_stored_ecs.lua Check_that_SDL_returns_empty_array_no_appID_no_stored_ecs
```

Comment:
@maria-mih, base on clarificatios, plase add additional cases when:
app with appID is not registered at all.
app with appID is unregistered.